spec/models/organization_spec.rb

```diff
@@ -339,4 +339,18 @@
       expect(organization.enough_credits?(1)).to be(true)
     end
   end
+
+  describe "#public_articles_count" do
+    it "returns the count of published articles" do
+      published_articles = create_list(:article, 3, organization: organization, published: true)
```

Comment:
Minor: this test would succeed even if the `published` scope wasn't called, making it a potential false-positive. Maybe create two published posts and one unpublished post?